huey/storage.py

```diff
@@ -104,9 +104,18 @@ def flush_all(self):
 
 
 class RedisStorage(BaseStorage):
-    def __init__(self, name='huey', blocking=False, read_timeout=1,
-                 max_errors=1000, connection_pool=None, **connection_params):
-        if connection_pool is None:
+    def __init__(self, name='huey', blocking=False, read_timeout=1, max_errors=1000,
```

Comment:
For future reference please note that peewee uses a 79 character line limit. I will go ahead and fix it once it's merged.